sample/shared/compose/src/commonMain/kotlin/com/arkivanov/sample/shared/dialog/DialogContent.kt

```diff
@@ -0,0 +1,31 @@
+package com.arkivanov.sample.shared.dialog
+
+import androidx.compose.material.Text
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+
+@Composable
+fun DialogContent(dialogComponent: DialogComponent) {
+    AlertDialog(
```

Comment:
This code is shared between Android and Desktop. Unfortunately by default AlertDialog's width is pretty short on desktop. Could you please add `modifier = Modifier.width(300.dp)` here? It should make the dialog looking fine on both platforms.

Reply:
fixed